### PR TITLE
Add Mastery Check Tab

### LINKED ISSUE #232: Add Mastery Check Tab
As a user i want to view all the mastery checks under one tab.

### PR DESCRIPTION
**Pull request by ngzhenteng**
Enable switching of tabs when consultation and mc view/add commands are inputted.
Fixes #232

### CODE REVIEW

**Review by wilinetan — COMMENTED** [review]
Thanks for adding Mastery Check to the GUI!

File: src/main/java/seedu/address/logic/commands/add/AddCommand.java [hunk 1]
```diff
@@ -158,7 +158,8 @@ public CommandResult execute(Model model) throws CommandException {
             }
 
             model.addConsultation(toAddConsultation);
-            return new CommandResult(String.format(MESSAGE_SUCCESS_CONSULTATION, toAddConsultation));
+            return new CommandResult(String.format(MESSAGE_SUCCESS_CONSULTATION, toAddConsultation),
+                    CommandTargetFeature.Consultations);
```
Comment (Asuraxsoul) [inline src/main/java/seedu/address/logic/commands/add/AddCommand.java:162]: thanks for implementing switching of tabs to consultation for add consultation command

File: src/main/java/seedu/address/ui/MasteryCheckCard.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
+package seedu.address.ui;
```
Comment (Asuraxsoul) [inline src/main/java/seedu/address/ui/MasteryCheckCard.java:1]: thanks for adding in Mastery Check tab in GUI

**Review by Asuraxsoul — COMMENTED** [review]
LGTM

**Review by peironggg — COMMENTED** [review]
Thanks for making these GUI changes 👍

**Pull request merged**